maintainers/maintainer-list.nix

```diff
@@ -10357,6 +10357,18 @@
     githubId = 1476865;
     name = "jigglycrumb";
   };
+  jiriks74 = {
```

Comment:
Oh and this should be a separate commit

Reply:
Fixed in b489ea0f4e1904db70c37bfbfe481fa2cc84c9f3